logstash-core/src/main/java/org/logstash/ext/JrubyTimestampExtLibrary.java

```diff
@@ -254,5 +248,9 @@ public IRubyObject ruby_year(ThreadContext context)
         {
             return RubyFixnum.newFixnum(context.runtime, this.timestamp.getTime().getYear());
         }
+
+        private static RubyTimestamp fromRString(final Ruby runtime, final RubyString string) {
+            return RubyTimestamp.newRubyTimestamp(runtime, new Timestamp(string.toString()));
```

Comment:
@jakelandis this is only called when guarded by an `instanceof` on `string` upstream => jup, `string` is never `null`.